modules/customize-certificates-replace-default-router.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="replacing-default-ingress-{context}"]
+= Replacing the default ingress certificate
+
+You can replace the default ingress certificate for all
+applications under the `.apps` subdomain. After you replace
+the certificate, all applications, including the web console
+and CLI, will have encryption provided by specified certificate.
+
+.Prerequisites
+
+* You must have a wildcard certificate, in the PEM format, for use. 
```

Comment:
wildcard certificate and its private key, both in the PEM format

Reply:
Updated to use this verbiage.